Week1/homework/style.css

```diff
@@ -1 +1,37 @@
-/* add your styling here */
```

Comment:
The look for your application on mobile can be improved:

![screenshot 2019-01-11 at 18 27 56](https://user-images.githubusercontent.com/2788771/51050066-e53e7500-15cf-11e9-8a8f-0fa9a53b0dae.png)